### ISSUE TITLE
Pre-trained models - no means or labels files

### ISSUE THREAD

**Issue by zeryx** [Microsoft/CNTK#2326]
Hey guys, I'm writing some documentation about how to use CNTK in a production environment with an inference function using a pre-trained model but I realized there are no label or means files for any of the available models.
If someone could clean this up that would be great, thanks!

**Comment by cha-zhang** [comment]
All CNTK trained model are trained with scripts in the examples and they are clearly stated: 
https://github.com/Microsoft/CNTK/blob/master/PretrainedModels/Image.md 
You may find the mean file information in the corresponding training script.

**Comment by zeryx** [comment]
Hey @cha-zhang thanks for taking a look at this. The trainingdescription files do not reference absolute paths or filenames themselves as they are set by relative paths, which unfortunately are quite difficult to trace back without intimate knowledge of the CNTK code base.
It'd be really appreciated if you could point me in the right direction to where those files are located, it would also be incredibly helpful to explicitly state where those files are for each model described in the `Image.md` docs.

**Comment by cha-zhang** [comment]
Let me use ResNet model as an example: 
https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Classification/ResNet/BrainScript/ResNet18_ImageNet1K.cntk 

Note the mean file is ImageNet1K_mean.xml". The mean file is here: 
https://github.com/Microsoft/CNTK/tree/master/Examples/Image/DataSets/ImageNet 

Another example: 
https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Classification/AlexNet/BrainScript/AlexNet_ImageNet.cntk 

This one doesn't use a mean file. 
```
featNorm = features - Constant(114)
```
So RGB value all subtract constant value 114.

**Comment by zeryx** [comment]
Excellent thanks, That should help. Hover we're still missing the label files for each model / dataset type, and I can't seem to find a path to one anywhere.I played with a gist of an imagenet 1k labels json file, but it seems like there must be something wrong with the order as the probabilities don't seem accurate on some trial images I used.

**Comment by cha-zhang** [comment]
For ImageNet we should be using the standard label indices as everyone else.

**Comment by zeryx** [comment]
Excellent, in that case, can you link me the standard label indicies used? Or are the files lost and unrecoverable?
As I said the example file I found while searching online seems to not line up with the imageNet model I was working with. 
We're writing documentation for CNTK support and I'd really love to have a full working inference demo running on our platform before we officially release the tutorial, and this is blocking that release.

**Comment by cha-zhang** [comment]
I found this accidentally: 
https://github.com/Microsoft/CNTK/blob/master/Examples/Evaluation/UWPImageRecognition/App/Assets/imagenet1000_clsid.txt

**Comment by cha-zhang** [comment]
The issue should have been resolved. Close for now. Feel free to reopen.
